mm2src/komodefi_cli/src/cli_cmd_args/cmd_set_config.rs

```diff
@@ -1,34 +1,40 @@
+use crate::cli::get_cli_root;
 use crate::error_anyhow;
+
 use anyhow::{anyhow, Result};
 use clap::Args;
 use common::log::error;
 use inquire::{Confirm, Text};
 use std::fs::File;
 use std::io::Read;
 
+const DEFAULT_RPC_URL: &str = "127.0.0.1:7789";
```

Comment:
Isn't the default rpc url supposed to have `7783` port?

Reply:
for testing..reverted